spec/tides-data-package.json

```diff
@@ -0,0 +1,2382 @@
+                "{\n  \"path\": \"file.csv\"\n}\n",
+                "{\n  \"path\": \"http://example.com/file.csv\"\n}\n"
+              ],
+              "context": "Implementations need to negotiate the type of path provided, and dereference the data accordingly."
+            },
+            "title": {
+              "title": "Title",
+              "description": "A human-readable title.",
+              "type": "string",
+              "examples": [
+                "{\n  \"title\": \"My Package Title\"\n}\n"
+              ]
+            }
+          },
+          "context": "Use of this property does not imply that the person was the original creator of, or a contributor to, the data in the descriptor, but refers to the composition of the descriptor itself."
+        },
+        "context": "This property is not legally binding and does not guarantee that the package is licensed under the terms defined herein.",
+        "examples": [
+          "{\n  \"licenses\": [\n    {\n      \"name\": \"odc-pddl-1.0\",\n      \"path\": \"http://opendatacommons.org/licenses/pddl/\",\n      \"title\": \"Open Data Commons Public Domain Dedication and License v1.0\"\n    }\n  ]\n}\n"
+        ]
+      },
+      "resources": {
+        "propertyOrder": 100,
+        "title": "Tabular Data Resources",
+        "description": "Array of data files included in your package, formated as a tabular-data-resource",
+        "type": "array",
+        "minItems": 1,
+        "items": {
+          "title": "Tabular Data Resource",
+          "description": "A Tabular Data Resource.",
+          "type": "object",
+          "required": [
+              "name",
+              "path",
+              "schema"
+          ],
+          "properties": {
+            "profile": {
+              "enum": [
+                "tabular-data-resource"
+              ],
+              "propertyOrder": 10,
+              "title": "Profile",
+              "description": "Should be `tides-data-package`",
+              "context": "Every Package and Resource descriptor has a profile. The default profile, if none is declared, is `data-package` for Package and `data-resource` for Resource.",
+              "type": "string",
+              "examples": [
+                "{\n  \"profile\": \"tides-data-package\"\n}\n",
+                "{\n  \"profile\": \"http://example.com/my-profiles-json-schema.json\"\n}\n"
+              ]
+            },
+            "name": {
+              "propertyOrder": 20,
+              "title": "Name",
+              "description": "An identifier string. Lower case characters with `.`, `_`, `-` and `/` are allowed.",
+              "type": "string",
+              "pattern": "^([-a-z0-9._/])+$",
+              "context": "This is ideally a url-usable and human-readable name. Name `SHOULD` be invariant, meaning it `SHOULD NOT` change when its parent descriptor is updated.",
+              "examples": [
+                "{\n  \"name\": \"my-nice-name\"\n}\n"
```

Comment:
add actual example

Reply:
Resolved in [dd70956](https://github.com/TIDES-transit/TIDES/pull/124/commits/dd70956050e6e687c756f158467be77e78c926bc)